olive/passes/onnx/optimum_conversion.py

```diff
@@ -33,6 +34,7 @@ def _run_for_config(
         self, model: OptimumModel, data_root: str, config: Dict[str, Any], output_model_path: str
     ) -> Union[ONNXModel, CompositeOnnxModel]:
         assert len(model.model_components) > 0
+        assert model.model_file_format == ModelFileFormat.OPTIMUM
```

Comment:
>ModelFileFormat.OPTIMUM [](http://example.com/codeflow?start=42&length=23)

I don't see the model_file_format is used for OPTIMUM. 

Do we really need it?

Reply:
I didn't mean to touch this file actually, I forgot to revert t hose changes when I reverted the merge. I'll get rid of it.